Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -34,7 +34,7 @@ open class _ChatChannelListVC<ExtraData: ExtraDataTypes>: ViewController,
         return collection
     }()
     
-    public private(set) lazy var createNewChannelButton: ChatChannelCreateNewButton<ExtraData> = {
+    public private(set) lazy var createNewChannelButton: _ChatChannelCreateNewButton<ExtraData> = {
```

Comment:
This doesn't have to be `_ChatChannelCreateNewButton<ExtraData>` but it can be just `UIButton`, right? -> less public API surface area and more flexibility for users.